python/lsst/sconsUtils/scripts.py

```diff
@@ -195,7 +195,16 @@ def finish(defaultTargets=DEFAULT_TARGETS,
                       nfail=`find {0} -name \*.failed | wc -l | sed -e 's/ //g'`; \
                       if [ $$nfail -gt 0 ]; then \
                           echo "Failed test output:" >&2; \
-                          find {0} -name \*.failed -exec cat {{}} \; >&2; \
+                          for f in `find {0} -name \*.failed`; do \
```

Comment:
You mean the bit after the `do`? Probably not needed. It was left over from the edited line.